module/data/item/templates/ability.mjs

```diff
@@ -186,6 +186,26 @@ export default class AbilityTemplate extends ActionTemplate.mixin(
     return this.parent;
   }
 
+  async chooseTier( ) {
+    const promptFactory = PromptFactory.fromDocument( this.parent );
+    const tier = await promptFactory.getPrompt( "chooseTier" );
+
+    if ( !tier || tier === "cancel" || tier === "close" ) return;
+
+    const updatedItem = await this.parent.update( {
+      "system.tier": tier,
+    } );
+
+    if ( foundry.utils.isEmpty( updatedItem ) ) {
```

Comment:
Also feels like something we should make a function out of, we're starting to use it a lot. Something like `warnIfNoUpdate( warnMessageKey )`